folly/test/AHMIntStressTest.cpp

```diff
@@ -107,7 +107,8 @@ TEST(AHMIntStressTest, Test) {
   SCOPE_EXIT { delete objs; };
 
   std::vector<std::thread> threads;
-  for (int threadId = 0; threadId < 64; ++threadId) {
+  threads.reserve(64);
+for (int threadId = 0; threadId < 64; ++threadId) {
```

Comment:
This is an example of what I mean by the vector-reserve pass duplicating constants that really shouldn't be.

Reply:
Definitely not desirable, agreed.